.github/workflows/rust_checks.yml

```diff
@@ -43,7 +43,10 @@ jobs:
         run: cargo fmt --check --all
 
       - name: Run tests
-        run: cargo test -p zkevm-metrics -p witness-generator
+        env:
+          RPC_URL: ${{ secrets.RPC_URL }}
+          RPC_HEADERS: ${{ secrets.RPC_HEADERS }}
+        run: cargo test -p zkevm-metrics -p witness-generator -- --no-capture
```

Comment:
The `--no-capture` is for double-checking the test aren't skipped due to envs not properly set. I did a test of not setting the env vars to double check that all was running as expected.